.github/workflows/update-gov-zone-file.yml

```diff
@@ -1,7 +1,13 @@
-name: Update gov.txt from CZDS API zone file
-run-name: Update gov.txt from CZDS API zone file
+name: Update gov.txt from Cloudflare zone file
+run-name: Update gov.txt from Cloudflare zone file
 
 on:
+  pull_request:
```

Comment:
Temporary workflow triggers to demonstrate that the zone file properly when run on GitHub actions. Will remove this section and the following `current-federal.csv`, `current-full.csv`, and `gov.txt`, which are affected by running the workflow on this branch to test.

Reply:
Agree. Before merging, each of these documents _need_ to be removed from the PR – but I understand why gov.txt is here temporarily.